### ISSUE TITLE
批量预测与单幅图像预测结果差别很大

### ISSUE THREAD

**Issue by zyz207** [PaddlePaddle/Paddle#35540]
import paddle
from paddle.vision.models import resnet50
model=resnet50(pretrained=True)
model.eval()

x_test_=paddle.rand([1, 3, 224, 224])
x_test=paddle.concat([x_test_,x_test_,x_test_,x_test_],axis=0)
feature_data_ = model(x_test_).cpu().numpy()
feature_data = model(x_test).cpu().numpy()
print((abs(feature_data_[0]-feature_data[0])).sum())
打印结果是 0.6741117

PaddlePaddle版本：2.1
GPU Compute Capability: 8.6, Driver API Version: 11.4, Runtime API Version: 11.2
系统环境：Windows10，python3.8

**Comment by heavengate** [comment]
直接`.sum()=0.674`并不等于得出结果差很大的结论，有可能feature map本身数据很大，0.674只是一个微小的数值波动，建议比较相对值
